tsl/test/sql/include/cagg_ddl_common.sql

```diff
@@ -288,15 +288,9 @@ SELECT * FROM truncate_partitioned;
 -- ALTER TABLE tests
 \set ON_ERROR_STOP 0
 -- test a variety of ALTER TABLE statements
-ALTER TABLE :drop_chunks_mat_table_u RENAME chunk_id TO bad_name;
-ALTER TABLE :drop_chunks_mat_table_u ADD UNIQUE(chunk_id);
```

Comment:
why are you removing tests here? can't you do these tests on a different column instead of removing them?

Reply:
fixed!